packages/icebreaker/SOURCES/icebreaker.sgifixes.patch

```diff
@@ -0,0 +1,48 @@
+diff -Naur icebreaker-2.2.1.orig/themes.c icebreaker-2.2.1/themes.c
+--- icebreaker-2.2.1.orig/themes.c	2021-03-01 15:52:20.000000000 +0000
++++ icebreaker-2.2.1/themes.c	2022-02-25 22:15:58.002949004 +0000
+@@ -147,7 +147,7 @@
+ 	char themefilename[256]; // fix -- use defined OS constant (here and throughout this function)
+ 	char optbuf[21];
+ 	char valbuf[256];
+-	char loadfilebuf[256];
++	char loadfilebuf[512];
```

Comment:
🤔

Reply:
I just followed the errors, I'm assuming there's something dumb with the length of a (char) going on.